aspnet/2-structured-data/Models/DatastoreBookStore.cs

```diff
@@ -84,16 +69,16 @@ public static Property NewProperty(DateTime? value)
         /// [START toentity]
         public static Entity ToEntity(this Book book)
         {
+            // Other than the aforementioned ToKey() issues, this is really

```

Comment:
One advantage is that it means things like this end up as a single expression, which can then be used in an expression-bodied method. Personally I just like avoiding the repetition of `entity` :)